app/views/projects/versions/list.scala.html

```diff
@@ -91,9 +92,16 @@ <h4 class="panel-title pull-left">@messages("project.versions")</h2>
                                                         <span class="date">@prettifyDate(version.createdAt.get)</span>
                                                         @defining(version.downloadCount) { d: Int =>
                                                             <span class="date">
-                                                                @d Download@if(d != 1) { s } else { &nbsp; }
```

Comment:
This is a fix for something dumb I did in #289. Extra whitespace was never needed.